server/chat-plugins/roomsettings.js

```diff
@@ -349,7 +349,7 @@ exports.commands = {
 			this.addModAction(`${user.name} set modjoin to autoconfirmed.`);
 			this.modlog('MODJOIN', null, 'autoconfirmed');
 		} else if (target in Config.groups || target === 'trusted') {
-			if (room.battle && !user.can('makeroom') && target !== '+') return this.errorReply(`/modjoin - Access denied from setting modjoin past + in battles.`);
+			if (room.battle && !user.can('makeroom') && (target !== '+' || target !== '%')) return this.errorReply(`/modjoin - Access denied from setting modjoin past % in battles.`);
```

Comment:
This should be `&&`, not `||`.

`(target !== '+' || target !== '%')` always evaluates to `true`. :p

Reply:
Fortunately irrelevant, because there's a better way to write this:

```suggestion
			if (room.battle && !user.can('makeroom') && !'+%'.includes(target)) {
				return this.errorReply(`/modjoin - Access denied from setting modjoin past % in battles.`);
			}
```